src/components/settings/EditOrganizationInformationsDialog.tsx

```diff
@@ -14,20 +14,19 @@ import {
   useUpdateOrganizationInformationsMutation,
 } from '~/generated/graphql'
 import { theme } from '~/styles'
-import CountryCodes from '~/public/countryCodes.json'
+import { CountryCodes } from '~/countryCodes'
 import { addToast } from '~/core/apolloClient'
 
 const FILE_MAX_SIZE = 800000
 
-const countryData: { value: string; label: string }[] = Object.keys(CountryCodes).map(
-  (countryKey) => {
-    return {
-      value: countryKey,
-      // @ts-ignore
-      label: CountryCodes[countryKey],
-    }
+const countryData: { value: string; label: string }[] = (
+  Object.keys(CountryCodes) as Array<keyof typeof CountryCodes>
```

Comment:
This is used several times, maybe we can export it form the countryCode file
Also, the the type is just an array of strings : `Object.keys(CountryCodes) as string[]`

Reply:
Done! thanks for pointing this